includes/admin/payments/orders.php

```diff
@@ -602,59 +668,82 @@ function edd_order_details_adjustments( $order ) {
+					</div>
 					</li>
 				</ul>
-
+			</div>
+			<div class="edd-form-group">
 				<ul>
 					<li class="discount" style="display: none;">
+						<label for="edd_order_add_discount_select" class="edd-form-group__label screen-reader-text"><?php esc_html_e( 'Discounts', 'easy-ditigal-downloads' ); ?></label>
+					<div class="edd-form-group__control">
 						<?php
-						$d = edd_get_discounts( array(
-							'fields' => array( 'code', 'name' ),
-							'number' => 100,
-						) );
+						$d = edd_get_discounts(
+							array(
+								'fields' => array( 'code', 'name' ),
+								'number' => 100,
+							)
+						);
 
 						$discounts = array();
 
 						foreach ( $d as $discount_data ) {
 							$discounts[ $discount_data->code ] = esc_html( $discount_data->name );
 						}
-
-						echo EDD()->html->discount_dropdown( array(
-							'name'             => 'edd-order-add-discount-select',
-							'class'            => 'edd-order-add-discount-select',
-							'id'               => 'edd-order-add-discount-select',
-							'chosen'           => true,
-							'selected'         => false,
-							'show_option_all'  => false,
-							'show_option_none' => false,
-						) );  // WPCS: XSS ok.
+						echo EDD()->html->discount_dropdown(
+							array(
+								'name'             => 'edd-order-add-discount-select',
+								'class'            => 'edd-order-add-discount-select edd-form-group__input',
+								'id'               => 'edd-order-add-discount-select',
+								'chosen'           => true,
+								'selected'         => false,
+								'show_option_all'  => false,
+								'show_option_none' => false,
+							)
+						);  // WPCS: XSS ok.
 						?>
+					</div>
 					</li>
 				</ul>
-
+			</div>
+			<div class="edd-form-group">
 				<ul>
+				<div class="edd-form-group">
 					<li class="credit" style="display: none;">
-						<input type="text" class="edd-add-order-credit-description" value="" placeholder="<?php echo esc_attr( 'Description', 'easy-digital-downloads' ); ?>" />
-						<input type="text" class="edd-add-order-credit-amount" value="" min="1" placeholder="<?php echo esc_attr( 'Amount', 'easy-digital-downloads' ); ?>" />
+						<label for="edd_add_order_credit_description" class="screen-reader-text"><?php echo esc_html_e( 'Description', 'easy-digital-downloads' ); ?></label>
+					<div class="edd-form-group__control">
+						<input type="text" class="edd-add-order-credit-description" value="" id="edd_add_order_credit_description" placeholder="<?php echo esc_attr( 'Description', 'easy-digital-downloads' ); ?>" />
+				</div>
+				<div class="edd-form-group">
+						<label for="edd_add_order_credit_amount" class="screen-reader-text"><?php echo esc_html_e( 'Amount', 'easy-digital-downloads' ); ?></label>
+					<div class="edd-form-group__control">
+						<input type="text" class="edd-add-order-credit-amount" value="" id="edd_add_order_credit_amount" min="1" placeholder="<?php echo esc_attr( 'Amount', 'easy-digital-downloads' ); ?>" />
```

Comment:
Should this be a number input field instead of text? Since there is text I am assuming/hoping there is numerical validation going on during sanitization, but this should be a number, shouldn't it? (I know this was inherited.)

Reply:
This makes sense as a number, since it's already got the min attribute.